skyportal/tests/api/test_shifts.py

```diff
@@ -0,0 +1,22 @@
+from skyportal.tests import api
+
+
+def test_shift(public_group, super_admin_token, super_admin_user):
```

Comment:
Should be testing the deletion of a shift both with and without proper permissions.

Reply:
added